RawSpeed/NikonDecompressor.cpp

```diff
@@ -70,6 +67,10 @@ void NikonDecompressor::DecompressNikon(ByteStream *metadata, uint32 w, uint32 h
   pUp2[0] = metadata->getShort();
   pUp2[1] = metadata->getShort();
 
+  vector<ushort16> curve(65536);
+  for (size_t i = 0; i < curve.size(); i++)
```

Comment:
Be sure to test such things on as much of the raw files that use this codepath as you possibly can.